addonscript/test/test_task.py

```diff
@@ -0,0 +1,106 @@
+import os
+
+import pytest
+
+from addonscript.test import tmpdir
+from scriptworker.context import Context
+from scriptworker.exceptions import TaskVerificationError
+
+import addonscript.task as task
+
+assert tmpdir  # silence flake8
+
+
+@pytest.fixture(scope='function')
+async def context(tmpdir):
+    context = Context()
+    context.config = {
```

Comment:
Note: This was modified to be per-scope at https://github.com/mozilla-releng/addonscript/pull/3/files#diff-b0c91b03e78f0f3123febdaba5fdbf7aR18